entrypoint.sh

```diff
@@ -74,8 +84,9 @@ _push_to_github() {
         else
              git push origin
         fi
-
     else
+        echo "::debug::Push commit to remote branch $INPUT_BRANCH"
+        git pull origin "HEAD:$INPUT_BRANCH"
```

Comment:
I'm confused why this is pulling. What's the necessity to fetch stuff? And won't the merge caused by the pull cause problems?

The push below also needs updated. I'd expect it to be something like:
`git push origin "HEAD:/refs/heads/$INPUT_BRANCH"`, right?

Reply:
Ignore this comment. I didn't quite understand the purpose until I realized the pull was a typo. But I wasn't able to delete the line-item comment from the review :( sorry